pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -256,6 +261,35 @@ func validateTarget(vmClone *clonev1alpha1.VirtualMachineClone) []metav1.StatusC
 	return causes
 }
 
+func validateNewMacAddresses(vmClone *clonev1alpha1.VirtualMachineClone) []metav1.StatusCause {
```

Comment:
I may have been misunderstood, so I posted a PR to explain myself better: https://github.com/kubevirt/kubevirt/pull/12022

Can you please rebase over it and see if it answers your need?

Reply:
OK,I will do it

Reply:
@EdDev I have finsihed it and using the `link.ValidateMacAddress()`. So could you have a look?

Reply:
done